src/plugins/intel_npu/src/plugin/npuw/partitioning/patterns/opt.cpp

```diff
@@ -1271,8 +1305,8 @@ HostGatherDQ::HostGatherDQ(Context::Ref ctx) {
             auto matched_qcoeff = std::static_pointer_cast<ov::op::v0::Parameter>(matched_node_qcoeff);
             auto matched_ids = std::static_pointer_cast<ov::op::v0::Parameter>(matched_node_ids);
 
-            auto fp16vocab = ctx.get().unpack(matched_qweight, matched_qcoeff, ov::element::f16);
-            auto new_param = ctx.get().host_gather(fp16vocab, matched_ids);
+            auto newvocab = ctx.get().unpack(matched_qweight, matched_qcoeff, ov::element::f16);
+            auto new_param = ctx.get().host_gather(newvocab, matched_ids);
```

Comment:
This change doesn't really change anything except a rename, is that correct?

Reply:
Right, reverted